### DISCUSSION TITLE
"tab completion" / IntelliSense in markdown files?

### DISCUSSION THREAD

**Discussion by nshiab** [observablehq/framework#1491] [General]
Hi!

Is there a way to have "tab completion" / IntelliSense with VS Code in markdown files?

For example, if I start typing "Fil", it would be great to have a suggestion for "FileAttachment".

Thanks!

**Comment by chen56** [comment]
I have a similar question, too. It's completely text - input right now. How to configure code intellisense  - VSCode/IDEA... ?

**Comment by mbostock** [comment]
Please upvote #882 if you are interested in this feature.
